src/pages/availability-panel.tsx

```diff
@@ -36,12 +37,17 @@ const AvailabilityPanel: FC = () => {
     };
     const fetchIdleUsers = async () => {
       try {
-        const url = `${process.env.NEXT_PUBLIC_BASE_URL}/members/idle`;
+        const url = `${process.env.NEXT_PUBLIC_BASE_URL}/users/status?state=IDLE`;
         const { requestPromise } = fetch({ url });
         const fetchPromise = await requestPromise;
-        const { idleMemberUserNames } = fetchPromise.data;
-        const filterMembers = idleMemberUserNames.filter((username: string) => username);
-        const sortedIdleMembers = filterMembers.sort();
+        const { allUserStatus: idleUsers }: { allUserStatus: IdleUser[] } =
+          fetchPromise.data;
+
+        // Extract usernames from the idleUsers and sorting them in alphabetical order.
+        const sortedIdleMembers = idleUsers
+          .map((user) => user.username)
+          .sort((a, b) => a.localeCompare(b));
```

Comment:
❓ Why was this extra logic implemented? Do we not want to continue sorting users by most idle first, as returned by the API?(Just like on [idle-users page](https://status.realdevsquad.com/idle-users)?)

Reply:
The current [availability panel](https://status.realdevsquad.com/availability-panel) has users sorted in alphabetical order of their username, I was assuming that it was a requirement and applied the same logic here. 

I'll remove the extra logic and use the data returned by the API as is to show most idle users first.